src/DateTimePicker/__tests__/__snapshots__/DateTimePicker.test.js.snap

```diff
@@ -85,6 +85,7 @@ exports[`<DateTimePicker /> properly renders a DateTimePicker 1`] = `
     onNavigateBack={[Function]}
     onNavigateForward={[Function]}
     onOpen={[Function]}
+    renderView={[Function]}
```

Comment:
These snapshot changes came from updating `react-datetime`.  Functionality appears to be intact.